src/dataflow-types/src/client.rs

```diff
@@ -169,7 +170,14 @@ impl RustType<ProtoPeek> for Peek {
 
 /// Commands related to the computation and maintenance of views.
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-pub enum ComputeCommand<T = mz_repr::Timestamp> {
+pub struct ComputeCommand<T = mz_repr::Timestamp> {
+    pub cmd: ComputeCommandType<T>,
+    pub otel_ctx: OpenTelemetryContext,
```

Comment:
These fields and in particular the second one require comments. It's very unclear what properties it needs to have. Also nit but the first argument is not a "Type" it is the actual command. Imo maybe makes more sense / is less invasive to ship around a `TelemetriedComputeCommand` which wraps the command with the telemetry context. Afaict so far, none of the compute logic requires the telemetry context, and it is more of a wrapped command than a new command with more rich internals.

Reply:
:+1: about the naming suggestion.